packages/vitest/src/node/cli.ts

```diff
@@ -20,20 +21,54 @@ cli
+  .option('--coverage.all', 'Whether to include all files, including the untested ones into report. This option is not available for custom providers (default: false)')
+  .option('--coverage.clean', 'Clean coverage results before running tests. This option is not available for custom providers (default: true)')
+  .option('--coverage.cleanOnRerun', 'Clean coverage report on watch rerun. This option is not available for custom providers (default: true)')
+  .option('--coverage.reportsDirectory', 'Directory to write coverage report to. This option is not available for custom providers (default: ./coverage)')
+  .option('--coverage.reporter', 'Coverage reporters to use. Visit https://vitest.dev/config/#coverage-reporter for more information. This option is not available for custom providers (default: ["text", "html", "clover", "json"])')
+  .option('--coverage.reportOnFailure', 'Generate coverage report even when tests fail. This option is not available for custom providers (default: false)')
+  .option('--coverage.allowExternal', 'Collect coverage of files outside the project root. This option is not available for custom providers (default: false)')
+  .option('--coverage.skipFull', 'Do not show files with 100% statement, branch, and function coverage. This option is not available for custom providers (default: false)')
+  .option('--coverage.perFile', 'Check thresholds per file. See --coverage.lines, --coverage.functions, --coverage.branches and --coverage.statements for the actual thresholds. This option is not available for custom providers (default: false)')
+  .option('--coverage.thresholdAutoUpdate', 'Update threshold values: "lines", "functions", "branches" and "statements" to configuration file when current coverage is above the configured thresholds. This option is not available for custom providers (default: false)')
+  .option('--coverage.lines', 'Threshold for lines. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers')
+  .option('--coverage.functions', 'Threshold for functions. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers')
+  .option('--coverage.branches', 'Threshold for branches. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers')
+  .option('--coverage.statements', 'Threshold for statements. Visit https://github.com/istanbuljs/nyc#coverage-thresholds for more information. This option is not available for custom providers')
+  .option('--coverage.100', 'Shortcut to set all coverage thresholds to 100. This option is only available for the v8 provider (default: false)')
```

Comment:
There options were changed in https://github.com/vitest-dev/vitest/pull/4442. It also affects the title of this PR.

Reply:
Are they good now?